source/_patterns/basics/forms/disabled-states.hbs

```diff
@@ -0,0 +1,7 @@
+  {{> basics-form-group formLabel="Email" formInputType="email"
```

Comment:
disabled is set to true `formDisabled=true` it should add it to the input.
http://ec2-35-167-207-87.us-west-2.compute.amazonaws.com:8082/?p=basics-disabled-states